MapboxVision/Core/Telemetry.swift

```diff
@@ -1,30 +1,24 @@
 import Foundation
+import MapboxMobileEvents
 
-typealias TelemetryFileMetadata = [String: String]
+final class Telemetry: NSObject, TelemetryInterface {
+    private let manager = MMEEventsManager()
```

Comment:
shouldn't events manager be initialized with an access token?

Reply:
Yeah, init was lost in the process. Thanks